src/proto/message_translation/ssl_geometry.cpp

```diff
@@ -326,7 +326,7 @@ std::unique_ptr<SSLProto::SSL_GeometryData> createGeometryData(const Field& fiel
     return geometry_data;
 }
 
-std::optional<Field> createField(const SSLProto::SSL_GeometryData& geometry_packet)
+std::optional<Field> createFieldProto(const SSLProto::SSL_GeometryData& geometry_packet)
```

Comment:
This is returning a C++ Field object -> so `createField` is correct here.

There are other functions that actually need to be renamed in this file because the are generating Proto classes:
- `createVector2fProto`
- `createFieldLineSegmentProto`
- `createFieldCircularArcProto`
- `createGeometryFieldSizeProto`